lib/modules/core/capability_statement_sequence.rb

```diff
@@ -127,6 +127,10 @@ def json_formats
         rescue StandardError
           assert false, 'Capability Statement could not be parsed.'
         end
+
+        issues = Inferno::RESOURCE_VALIDATOR.validate(@conformance, versioned_resource_class)
```

Comment:
Would this make more sense in the `assert_valid_conformance` method? Or is this out of scope for that?

Reply:
I initially tried that, but `assert_valid_conformance` is used in the remaining conformance tests to check that we have received a CapabilityStatement that we can parse. I don't think we want all of those tests failing if we receive an invalid CapabilityStatement. It probably makes the most sense to rename that method to something like `assert_conformance_received`.

Reply:
That makes sense to me. Up to you whether you want to rename the method; I'm good approving it as-is.